### ISSUE TITLE
Extra messages about mamba being printed each time a terminal is opened

### ISSUE THREAD

**Issue by yuvipanda** [jupyterhub/repo2docker#1132]
### Bug description

When I open the terminal on a repo2docker built image, I'm getting the following output by default:

<img width="1073" alt="image" src="https://user-images.githubusercontent.com/30430/151796627-457e1772-da64-49af-8a1c-f68917cf5322.png">

#### Expected behaviour

Just a terminal, without any extra prompts and messages printed.

**Comment by manics** [comment]
It looks like `~/.bashrc` is updated everytime you open a terminal
```
jovyan@jupyter-manics-2d60f4eb81-2d31d9a52ef1fa2e6-2dkiggu130:~$ stat ~/.bashrc 
  File: /home/jovyan/.bashrc
  Size: 4305            Blocks: 16         IO Block: 4096   regular file
Device: 100085h/1048709d        Inode: 857612      Links: 1
Access: (0644/-rw-r--r--)  Uid: ( 1000/  jovyan)   Gid: ( 1000/  jovyan)
Access: 2022-01-31 21:02:01.168665302 +0000
Modify: 2022-01-31 21:02:01.148664980 +0000
Change: 2022-01-31 21:02:01.148664980 +0000
 Birth: -

jovyan@jupyter-manics-2d60f4eb81-2d31d9a52ef1fa2e6-2dkiggu130:~$ stat ~/.bashrc 
  File: /home/jovyan/.bashrc
  Size: 4305            Blocks: 16         IO Block: 4096   regular file
Device: 100085h/1048709d        Inode: 857612      Links: 1
Access: (0644/-rw-r--r--)  Uid: ( 1000/  jovyan)   Gid: ( 1000/  jovyan)
Access: 2022-01-31 21:02:27.597088922 +0000
Modify: 2022-01-31 21:02:27.581088667 +0000
Change: 2022-01-31 21:02:27.581088667 +0000
 Birth: -

jovyan@jupyter-manics-2d60f4eb81-2d31d9a52ef1fa2e6-2dkiggu130:~$ stat ~/.bashrc 
  File: /home/jovyan/.bashrc
  Size: 4305            Blocks: 16         IO Block: 4096   regular file
Device: 100085h/1048709d        Inode: 857612      Links: 1
Access: (0644/-rw-r--r--)  Uid: ( 1000/  jovyan)   Gid: ( 1000/  jovyan)
Access: 2022-01-31 21:02:37.217242139 +0000
Modify: 2022-01-31 21:02:37.189241693 +0000
Change: 2022-01-31 21:02:37.189241693 +0000
 Birth: -
```
Though `~/.bashrc` looks like it already contains the required snippet:
```
jovyan@jupyter-manics-2d60f4eb81-2d31d9a52ef1fa2e6-2dkiggu130:~$ tail -n17 ~/.bashrc 

# >>> mamba initialize >>>
# !! Contents within this block are managed by 'mamba init' !!
export MAMBA_EXE="/tmp/bin/micromamba";
export MAMBA_ROOT_PREFIX="/srv/conda";
__mamba_setup="$('/tmp/bin/micromamba' shell hook --shell bash --prefix '/srv/conda' 2> /dev/null)"
if [ $? -eq 0 ]; then
    eval "$__mamba_setup"
else
    if [ -f "/srv/conda/etc/profile.d/mamba.sh" ]; then
        . "/srv/conda/etc/profile.d/mamba.sh"
    else
        export PATH="/srv/conda/bin:$PATH"
    fi
fi
unset __mamba_setup
# <<< mamba initialize <<<
jovyan@jupyter-manics-2d60f4eb81-2d31d9a52ef1fa2e6-2dkiggu130:~$ 
```

**Comment by manics** [comment]
Maybe this is fixed by https://github.com/jupyterhub/repo2docker/pull/1128 which moves `micromamba` out of `tmp`?

**Comment by manics** [comment]
Maybe it's a bug in `mamba init`? @wolfv @SylvainCorlay ?

**Comment by manics** [comment]
https://github.com/jupyterhub/repo2docker/pull/1062/files#diff-b1c0ed3a44adc2328e4f7123eb109af31db2af06dc2787d9e59382019dede86c
added
https://github.com/jupyterhub/repo2docker/blob/8c21b9640119a2fe63c042f5dc5b873df1dc9029/repo2docker/buildpacks/conda/activate-conda.sh#L2
to `/etc/profile.d/activate-conda.sh`

Previously there wasn't a `conda shell init` step.

**Comment by wolfv** [comment]
Yeah, micromamba shell init only needs to be run once to produce the `.bashrc` -- as long as that's persisted and micromamba is not moved on the disk, it should be fine.
Is it possible to only run shell init once?
Otherwise we can add a quiet mode or you can add `> /dev/null` to hide output.

**Comment by minrk** [comment]
Yeah, I'm pretty sure this is fixed by #1128

**Comment by minrk** [comment]
Yup, fixed by #1128
